src/VisualStudio/Core/Def/Implementation/ProjectSystem/VisualStudioCompilationOutputFiles.cs

```diff
@@ -0,0 +1,73 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+using System;
+using System.IO;
+using System.Linq;
+using System.Reflection.PortableExecutable;
+using Microsoft.CodeAnalysis.Emit;
+using Roslyn.Utilities;
+
+namespace Microsoft.VisualStudio.LanguageServices.Implementation.ProjectSystem
+{
+    /// <summary>
+    /// Provides access to compilation outputs based only on the path of the output asssembly.
+    /// If PDB path is known upfront use <see cref="CompilationOutputFiles"/> instead.
+    /// </summary>
+    internal sealed class VisualStudioCompilationOutputFiles : CompilationOutputs
```

Comment:
So there's nothing really VisualStudio about this? Better name maybe "PdbGuessingCompilationOutputFiles" or something?

Reply:
Yeah, i couldn't figure a better name. It's Visual Studio specific hackery :)
Maybe `CompilationOutputFilesWithImplicitSymbols`?

Reply:
Renamed